projects/1.18/assets/1UNKNOWN/exblades/lang/zh_cn.json

```
@@ -0,0 +1,188 @@
+{
+  "exblades.name": "拔刀剑 秘传",
+  "exblades.desc": "追加刀剑 秘剑的复制品",
+
+  "exblades.category.mods": "附属模组",
+  "exblades.category.mods.desc": "致敬原作者",
+
+  "exblades.category.animations": "动画作品",
+  "exblades.category.animations.desc": "热门动画登场刀",
+
+  "exblades.category.games": "游戏作品",
+  "exblades.category.games.desc": "热门游戏登场刀",
+
+  "exblades.category.pso": "梦幻之星OL2",
+  "exblades.category.pso.desc": "梦幻之星OL2登场刀",
+
+  "exblades.category.mh": "怪物猎人",
+  "exblades.category.mh.desc": "游戏内登场刀",
+
+  "exblades.category.streamers": "主播",
+  "exblades.category.streamers.desc": "主播专属刀",
+  
+  "item.slashblade.ex.fluorescentbar": "荧光灯管",
+  "item.slashblade.ex.fluorescentbar.desc": "于星间穿梭时随兴制作的刀\n与原型不同，十分坚固",
+  "item.slashblade.ex.toyako": "洞爷湖",
+  "item.slashblade.ex.toyako.desc": "白夜叉",
+
+  "item.slashblade.ex.wanderer": "风来剑",
+  "item.slashblade.ex.wanderer.desc": "沙漠魔域之剑\n一度埋没的诞生之路\n今由后人发掘重见天日",
+  "item.slashblade.ex.wanderer.material": "刀组件「风来」",
+
+  "item.slashblade.ex.laemmle": "次元拔刀",
+  "item.slashblade.ex.laemmle.desc": "仿造某炼金术士剑客所持有之刀的所得之物",
+  "item.slashblade.ex.laemmle.material": "刀组件「炼金」",
+
+  "item.slashblade.ex.terra": "泰拉刃",
+  "item.slashblade.ex.terra.desc": "自然之力铸就之刃的复现\n力量已不复往日",
+  "item.slashblade.ex.terra.material": "刀组件「泰拉」",
```

原文`刀剣の素材箱「大地」`
```suggestion
  "item.slashblade.ex.terra.material": "刀组件「大地」",
```
**请遵循原文**，此类问题请全文通查